met-api/src/met_api/services/poll_answers_service.py

```diff
@@ -0,0 +1,30 @@
+"""Service for PollAnswer management."""
+from http import HTTPStatus
+
+from sqlalchemy.exc import SQLAlchemyError
+from met_api.exceptions.business_exception import BusinessException
+from met_api.models.poll_answers import PollAnswer as PollAnswerModel
+
+
+class PollAnswerService:
+    """PollAnswer management service."""
+
+    @staticmethod
+    def get_poll_answer(poll_id):
+        """Get poll answer by poll id."""
+        poll_answer = PollAnswerModel.get_answers(poll_id)
+        return poll_answer
+
+    @staticmethod
+    def create_bulk_poll_answers(poll_id: int, answers_data: list):
+        """Bulk insert of poll answers."""
+        try:
+            if len(answers_data) > 0:
+                PollAnswerModel.bulk_insert_answers(poll_id, answers_data)
+        except SQLAlchemyError as e:
+            raise BusinessException(str(e), HTTPStatus.INTERNAL_SERVER_ERROR) from e
+
+    @staticmethod
+    def delete_poll_answers(poll_id: int):
```

Comment:
If the PO thinks we should allow polls to be changed while they're being voted on, then we may need to do a check to see if answers have been responded to and prevent deletion if so.